defaults/main.yml

```diff
@@ -0,0 +1,9 @@
+---
+rabbitmq:
+    user: "user"
+    password: "password"
+    plugins: []
+
+vagrant_local:
```

Comment:
This is a dependency between roles that should be fixed. But can be done in another pr